### ISSUE TITLE
Changes to update checking

### ISSUE THREAD

**Issue by gdude2002** [TeamJM/journeymap#7]
The famed citricsquid (the original owner of the Minecraft Forums) ended up on our Discord server today. He had a notification about how JM does update checking, so I'm posting it here to ensure that it doesn't get lost.

```
Hi, I noticed that journeymap uses widget.mcf.li for update checking: due to changes to 
the mods.curse.com/curseforge.com website, widget.mcf.li was unavailable for a week or
so. The service is back online under a new name now, with improved reliability. I tried to 
keep the API backwards compatible but there may be minor changes that have broken 
your implementation due to changes in the curseforge website. The new API endpoint 
for your project is: https://api.cfwidget.com/minecraft/mc-mods/journeymap (old URLs 
redirect to the new location). If there's any minor change to the API that has broken your 
implementation that I can address by adjusting the API output, please let me know, and
I can make changes. Thanks!
```

[![](https://dl.dropboxusercontent.com/s/mhcexkxcq4eouqc/DiscordCanary_2017-11-21_17-38-12.png)](https://dl.dropboxusercontent.com/s/mhcexkxcq4eouqc/DiscordCanary_2017-11-21_17-38-12.png)

**Comment by techbrew-mc** [comment]
Fixing in 5.6.0 branches

**Comment by techbrew-mc** [comment]
Fixed in 5.6.0b1
